assets/alertmanager/network-policy-downstream.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  annotations:
+    include.release.openshift.io/hypershift: "true"
+    include.release.openshift.io/ibm-cloud-managed: "true"
+    include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/single-node-developer: "true"
+  labels:
+    app.kubernetes.io/managed-by: cluster-monitoring-operator
+    app.kubernetes.io/part-of: openshift-monitoring
+  name: alertmanager-access
```

Comment:
do we need to add the `-access` because of a convention or sth?
I'd use the same name as the component CR name/Deployment name etc. So the mapping is easier for users.
If we want to add another one for that convention, we can think about naming